src/CaptureFileInfo/include/CaptureFileInfo/Manager.h

```diff
@@ -28,7 +32,8 @@ class Manager {
   // paths that use slash (/) as directory separators, are equal to paths that are using backslash.
   // TODO(http://b/218298681) use std::filesystem::equivalent instead of operator== to check whether
   // 2 paths are actually pointing to the same file.
-  void AddOrTouchCaptureFile(const std::filesystem::path& path);
+  void AddOrTouchCaptureFile(const std::filesystem::path& path,
+                             std::optional<absl::Duration> capture_length = std::nullopt);
```

Comment:
I would prefer to not use the default value `std::nullopt` here. The capture length should always be available and the user of this function should be forced to set it manually to std::nullopt when not available

Reply:
XD ah thanks for pointing this out. I was lazing and don't want to change all the tests that call `AddOrTouchCaptureFile` with only the path.

I will make the change accordingly.

Reply:
Does it need to be an optional?

Reply:
The `GetCaptureLengthFromPath` might return a `std::nullopt`. But I can also use kMissingCaptureLengthValue for this case. WDYT?